src/lib/third-party-tags/imr-worldwide.ts

```diff
@@ -67,22 +67,24 @@ const onLoad = (): void => {
 			? sectionFromMeta
 			: 'The Guardian - brand only';
 
-	const nolggGlobalParams = {
-		sfcode: 'dcr',
-		apid: subBrandApId,
-		apn: 'theguardian',
-	};
+	if (typeof subBrandApId === 'string') {
```

Comment:
I think you can avoid checking the type of `subBrandApId` here, since

```typescript
guMetadata[sectionFromMeta] || guMetadata['brand-only']
```

should always be defined (if we can't lookup the section in the object, we fall back to using the `brand-only` key, which is always present).

I think it'd be simpler to simply assert this yourself (e.g. add an `as string` to the declaration of `subBrandApId`).

 Or, possibly more complicated, to try make this explicit in the typeof `guMetadata`. I haven't tested this but something like:

```typescript
const guMetadata: {
	'brand-only': string;
	[key: string]: string | undefined;
} = {
  // ... the other values
  'brand-only': 'P0EE0F4F4-8D7C-4082-A2A4-82C84728DC59',
}
```

Reply:
Will do 👍

Reply:
Nice thanks!